bin/emulator/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+set(TARGET_NAME emulator_wrapper)
+set(SOURCES
+    wrapper.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/../assigner/src/utils.cpp)
```

Comment:
That looks weird, I'm not sure, what's going on here. Compiling sources from other directory - there is something wrong about it.

Reply:
Source files are not supposed to be reused in general. It may lead to linking errors, if libraries with the same source file are used together. Also it adds some efforts for building, we have to compile the file twice. So I think it's time to create a separate library with assigner utils. And we could add a runner function into it (either now or later in my PR with the test).

Reply:
Well, yep, I'm kinda agree with Misha. Especially if taking into account that we are producing a shared library here.

I guess we can survive this, but moving utils code just feels more solid. Like if we wanna reuse the code, then let's do it properly.

Reply:
I think I've never seen a `.cpp` file used more than once to compile different executables

Reply:
Btw all the utils for assigner are actually more look like evmc utils and data types utils. So probably there is a way to move them somewhere closer to evmc/data types.

Reply:
what about move implementations o `to_str` to `utils.hpp`?

Reply:
It can lead to the same linkage errors, or you have to make all such functions `inline`. Creating a library is the best way to reuse the code here. But I agree to merge it like this right now, I could create the library in my following PR.